libs/editor/WordPressEditor/build.gradle

```diff
@@ -71,7 +71,7 @@ dependencies {
     implementation 'org.apache.commons:commons-lang3:3.5'
     implementation 'com.android.volley:volley:1.1.1'
     implementation "org.wordpress:utils:$wordPressUtilsVersion"
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$gradle.ext.kotlinVersion"
     api ("$gradle.ext.aztecAndroidAztecPath") {
```

Comment:
Question (❓): I've seen the `aztecVersion` being added and removed as I was traversing the commits in this PR. I am not sure what would be the best, to have that or keep it as is, which is without. Maybe, having the `aztecVersion` around, just like it is done for the `aztecAndroidWordPressShortcodesPath` and `aztecAndroidWordPressCommentsPath` is best, even if only for consistency purposes. Wdyt?

`api ("$gradle.ext.aztecAndroidAztecPath:$aztecVersion") { ... }`

Reply:
Great, thanks for the explanation Oguz, I mentioned that just in case, I am ok with either making this change here or as a separate PR, which could maybe improve on all other such inconsistencies. All good from my side and noted! 👍